pandas_gbq/gbq.py

```diff
@@ -1032,17 +1032,30 @@ def to_gbq(dataframe, destination_table, project_id, chunksize=10000,
                                      "already exists. "
                                      "Change the if_exists parameter to "
                                      "append or replace data.")
-        elif if_exists == 'replace':
-            connector.delete_and_recreate_table(
-                dataset_id, table_id, table_schema)
-        elif if_exists == 'append':
-            if not connector.schema_is_subset(dataset_id,
-                                              table_id,
-                                              table_schema):
-                raise InvalidSchema("Please verify that the structure and "
-                                    "data types in the DataFrame match the "
-                                    "schema of the destination table.")
+        else:
+            delay = 0
+            if not connector.verify_schema(dataset_id, table_id, table_schema):
```

Comment:
This should be `if not connector.schema_is_subset(dataset_id, table_id, table_schema):`

See https://travis-ci.org/parthea/pandas-gbq/jobs/250715768:
The following unit test is failing :
```
pandas_gbq/tests/test_gbq.py::TestToGBQIntegrationWithServiceAccountKeyPath::test_upload_subset_columns_if_table_exists_append
```